gallery/src/styles/NxCard/NxCardHorizontalLayoutExample.tsx

```diff
@@ -8,93 +8,87 @@ import React from 'react';
 import { faShapes, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
 
 import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+const chart = require('./chart.png');
 
 export default function NxCardColumnLayoutExample() {
   return (
     <div className="nx-card-container nx-card-container--column">
-      <section className="nx-card" aria-label="card 1 in column layout">
+      <section className="nx-card nx-card--horizontal" aria-label="card 1 in column layout">
+        <div className="nx-card__call-out">
```

Comment:
If we do care at all about trying to keep the structure the same between the two types of cards, I think it would be possible using `display: contents`.  You could have them always structured the way the vertical cards are currently, and then horizontal cards would set `display: contents` on the `.nx-card__content` in order to position the callout and text independently.

Reply:
> If we do care at all about trying to keep the structure the same between the two types of cards, I think it would be possible using display: contents. You could have them always structured the way the vertical cards are currently, and then horizontal cards would set display: contents on the .nx-card__content in order to position the callout and text independently.

This would also avoid the currently situation where in horizontal cards, `.nx-card__content` and `.nx-card__text` are kind of redundant.

Reply:
> > If we do care at all about trying to keep the structure the same between the two types of cards, I think it would be possible using display: contents. You could have them always structured the way the vertical cards are currently, and then horizontal cards would set display: contents on the .nx-card__content in order to position the callout and text independently.
> 
> This would also avoid the currently situation where in horizontal cards, `.nx-card__content` and `.nx-card__text` are kind of redundant.

I did have that before, it caused some other (minor) issues. I still think that making the cards convert from one layout to another was a mistake.

Reply:
> I did have that before, it caused some other (minor) issues. I still think that making the cards convert from one layout to another was a mistake.

Could you elaborate on the issues you saw?  To me it isn't about the convertibility, it's about decreasing the mental load.  It would be easier on developers if the structure was the same between the two cards.

Reply:
> Could you elaborate on the issues you saw? To me it isn't about the convertibility, it's about decreasing the mental load. It would be easier on developers if the structure was the same between the two cards.

I've tried it myself and it seems to work.  Are there issues that I'm not noticing? See https://github.com/sonatype/sonatype-react-shared-components/pull/311

Reply:
> > Could you elaborate on the issues you saw? To me it isn't about the convertibility, it's about decreasing the mental load. It would be easier on developers if the structure was the same between the two cards.
> 
> I've tried it myself and it seems to work. Are there issues that I'm not noticing? See #311

No longer applicable as we have removed the horizontal layouts entirely.